src/compiler/axis.ts

```diff
@@ -85,9 +85,9 @@ export function grid(model: Model, channel: Channel) {
   }
 
   // If `grid` is unspecified, the default value is `true` for
-  // - X and Y that have (1) quantitative fields that are not binned or (2) time fields.
+  // - X and Y that have (1) quantitative fields that are not binned or (2) time fields that don't have a timeUnit.
```

Comment:
what if you have timeUnit but the timeUnit leads to linear scale?  (e.g., minutes / seconds)?